src/test/java/com/happeningnow/controller/ControllerOrganizerTest.java

```diff
@@ -0,0 +1,135 @@
+package com.happeningnow.controller;
+
+import com.happeningnow.model.Organizer;
+import com.happeningnow.repository.OrganizerRepository;
+import com.happeningnow.util.CustomPageImpl;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.ActiveProfiles;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.tuple;
+import static org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+
+
+@SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT)
+@ActiveProfiles("test")
+public class ControllerOrganizerTest {
+
+    @Autowired
+    private TestRestTemplate resTemplate;
+
+    @Autowired
+    private OrganizerRepository organizerRepository;
+
+    private Organizer organizer1;
+
+    private Organizer organizer2;
+
+    @BeforeEach
```

Comment:
write a after each and delete all data from the database, this will make sure that we don't have misleading results. and to be honest. in this case, do not use a before each, create a record in each method, so that we make sure to be testing a specific record.

Reply:
ok